homework.txt

```diff
@@ -0,0 +1,53 @@
+Question#1: Multiplication Tables * tried multiple solutions fo the 
+ bonus question but no chance ;( * 
+
+for ( var i =0 ;  i <=10 ; i++){
+
+console.log( i * 9 )
+}
+
+Question#2: * Couldn't get a correct output, and can't see where's the problem* 
+
+function assignGrade(grade){
+
+for (var i = 60 ; i <= 100; i++){
+
+  else if (i >= 90 ){
+
+console.log( " you Got an A "); 
+  }  else if (i >= 80  ){
+console.log( " you Got a B ");  }
+
+else  if (i >= 70 ){
+console.log( " you Got a C ");  }
+else if (i >= 60 ){
+console.log( " you Got an D ");
+
+  } else { console.log(" You dind't pass");
```

Comment:
You're in the right direction here, you'd probably were trying do something like 
```js
function assignGrade(grade) {
  if (i >= 90) {
    return " you Got an A.";
  } else if (i >= 80) {
    return " you Got a B.";
  } else if (i >= 70) {
    return " you Got a C.";
  } else if (i >= 60) {
    return " you Got an D.";
  } else {
    return " You didn't pass";
  }
}

for(var i = 60; i <= 100; i++){
	console.log("For " + i + assignGrade(i));
}
```

Let me know if you have more questions about this.